src/legacy/server/status/routes/api/register_stats.js

```diff
@@ -69,6 +69,11 @@ export function registerStatsApi(kbnServer, server, config) {
         if (isExtended) {
           const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('admin');
           const callCluster = (...args) => callWithRequest(req, ...args);
+          const collectorsReady = await collectorSet.areAllCollectorsReady();
+
+          if (shouldGetUsage && !collectorsReady) {
+            return boom.serverUnavailable('Not all collectors are ready.');
```

Comment:
Nit: collectors are an implementation detail that a consumer of the API might not be familiar with. Could we change this message to something more consumer-friendly about stats not being available at the moment, try again later?

Reply:
Also, seeing as this error message is repeated below as well, perhaps extract it into a const?